src/App.tsx

```diff
@@ -1,93 +1,113 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useCallback, useMemo } from 'react';
+import { useLocation } from 'react-router-dom';
+import { useLocalStorage } from './hooks/useLocalStorage';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Status } from './types/Status';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useLocalStorage('todos', []);
+
+  const location = useLocation();
+
+  const addTodo = (title: string) => {
```

Comment:
follow naming convention for handlers
```suggestion
  const handleAddTodo = (title: string) => {
```